data/test/language-snippets/format/list.sw

```diff
@@ -0,0 +1,306 @@
+
+/*******************************************************************/
+/*                       LIST ENCODING IN INT                      */
+/*                                                                 */
```

Comment:
Nice test case. 👍 😄

Reply:
Indeed!  This was the original source for #2105 in the first place - the formatter almost worked on list.sw except for one line.